README.md

```diff
@@ -38,27 +46,48 @@ class CdkStack extends cdk.Stack {
 | `nodeLayerVersion`   | Version of the Node.js Lambda layer to install, such as 29. Required if you are deploying at least one Lambda function written in Node.js and `addLayers` is true. Find the latest version number from [https://github.com/DataDog/datadog-lambda-js/releases][6].   |
-| `forwarderArn`       | When set, the plugin will automatically subscribe the functions' log groups to the Datadog Forwarder.                                                                                                                                                                |
+| `forwarderArn`         | When set, the plugin will automatically subscribe the functions' log groups to the Datadog Forwarder.  |
+| `flushMetricsToLogs` | Send custom metrics using Cloudwatch logs with the Datadog Forwarder Lambda function (recommended). Defaults to `true`. If you disable this parameter, it's required to set the parameters `site` and `apiKey` (or `apiKMSKey` if encrypted). |
+| `site`               | Set which Datadog site to send data, only needed when flushMetricsToLogs is `false`. Possible values are `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com` and `ddog-gov.com`. The default is `datadoghq.com`. |
+| `apiKey`             | Datadog API Key, only needed when `flushMetricsToLogs` is `false`. For more information about getting a Datadog API key, see the [API key documentation][8]. |
+| `apiKMSKey`          | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false`, and you are using KMS encryption. |
+| `enableDDTracing`    | Enable Datadog tracing on your Lambda functions. Defaults to `true`. When enabled, it's required to set the `forwarderARN` parameter. |
+| `injectLogContext`   | When set, the lambda layer will automatically patch console.log with Datadog's tracing ids. Defaults to `true`. |
 
-## How it works
+### AWS CDK Configurations
 
-The CDK construct takes in a list of lambda functions and installs the Datadog Lambda Library by attaching the Lambda Layers for [Node.js][2] and [Python][1] to your functions. It redirects to a replacement handler that initializes the Lambda Library without any required code changes.
+The AWS CDK library offers some additional options that you may include in your lambda functions and application / stack.
 
-[1]: https://github.com/DataDog/datadog-lambda-layer-python
-[2]: https://github.com/DataDog/datadog-lambda-layer-js
-[3]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-macros.html
-[4]: https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_core.Stack.html
-[5]: https://github.com/DataDog/datadog-lambda-python/releases
-[6]: https://github.com/DataDog/datadog-lambda-js/releases
-[7]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-subscriptionfilter.html
+### Tracing
+
+Enable X-Ray Tracing on your Lambda functions. For more information, see [CDK documentation][9].
```

Comment:
I think we need more details for customer's about if they should use x-ray or not. Maybe @Hesperide could double check the copy here.